ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -130,6 +130,11 @@ Pane {
         d.reload()
     }
 
+    function addTimestampToURL(url) {
```

Comment:
There are 2 similar functions. Can you please move it to qml utils (`/ui/imports/utils/Utils.qml` should be fine)?